src/context/token-repository.ts

```diff
@@ -26,6 +27,12 @@ export class TokenRepository extends BaseRepository<TokenEntity> {
           tokenIds
         });
       }
+
+      if (name) {
+        const nameLike = name.replace(/\*/g, "%");
+        filterQuery = filterQuery.andWhere(`${this.alias}.name LIKE :nameLike`, { nameLike });
```

Comment:
Let's optimize it a bit by checking if wildcard characters are present, if not we use equality operator, avoiding the overhead of a unnecessary `LIKE` operator

Reply:
Done.